solana-labs/wallet-adapter
# add coinbase wallet-adapter

# Summary
This PR adds a wallet adapter for Coinbase Wallet.  Solana is supported in Coinbase Wallet browser extension v.2.20 and higher.

# Testing

1. Ensure you have Coinbase Wallet browser extension v.2.20 or higher installed
2. Enable Experiment Mode from advanced settings. Go to Settings > Advanced Settings > Developer Mode
3. Add `CoinbaseWalletAdapter` to your dapps supported wallets in `WalletProvider`
4. Interact with the dapp using Coinbase Wallet
